### PR TITLE
Update role_copier

### LINKED ISSUE #57: Role Copier issue
Currently, it is unclear if the role copier can copy permanently dead people. It has been voted upon and decided that the role copier should not be allowed to copy dead people, and it shall be clarified.

### PR DESCRIPTION
**Pull request by Topkinsme**
This should fix #57

### CODE REVIEW

**Comment by 12Metatron** [comment]
there is one mention of the word dead which you forgot to change to respawning

**Comment by 12Metatron** [comment]
is it even possible for them to copy a solo role since those don't respawn?

**Comment by Topkinsme** [comment]
It currently cannot, but maybe if in the future it can, this will prevent there from being issues.

File: strong_power/role_copier [hunk 1]
```diff
@@ -2,5 +2,5 @@ Role Copier- (6)
 - Has the ability to copy the role of any respawning person.
 - Once copied, the role copier can choose to use the role as long as they wish. Each night they can use the role they have copied earlier, or try to copy a new role. The role copier will know the copied role immediately but cannot use it's powers till the next night (If it's a action). 
 - If the role copier attempts to copy any royalty role or any solo role, they instead get the warrior role.
-- Copying is instant. Other conditions are ported from the copied role. The copier cannot copy their dead teammates. The role copier cannot copy the same person twice in a game. The role copier loses their coped role when they die.
+- Copying is instant. Other conditions are ported from the copied role. The copier cannot copy from their teammates. The role copier cannot copy the same person twice in a game. The role copier loses their copied role when they die.
```
Comment (12Metatron) [inline strong_power/role_copier:5]: this should probably be split into multiple lines since they are about many different topics

Comment (Topkinsme) [reply strong_power/role_copier:5]: Will do

File: strong_power/role_copier [hunk 2]
```diff
@@ -2,5 +2,7 @@ Role Copier- (6)
 - Has the ability to copy the role of any respawning person.
 - Once copied, the role copier can choose to use the role as long as they wish. Each night they can use the role they have copied earlier, or try to copy a new role. The role copier will know the copied role immediately but cannot use it's powers till the next night (If it's a action). 
 - If the role copier attempts to copy any royalty role or any solo role, they instead get the warrior role.
```
Comment (12Metatron) [inline strong_power/role_copier:4]: can shorten to "any royalty or solo role"

Comment (Topkinsme) [reply strong_power/role_copier]: Fixed

**Comment by 12Metatron** [comment]
alright, You fixed everything. It's good now

**Comment by Topkinsme** [comment]
You can merge it then

**Pull request merged**